builds/e2e/templates/e2e-run.yaml

```diff
@@ -31,6 +31,7 @@ steps:
     E2E_EVENT_HUB_ENDPOINT: $(TestEventHubCompatibleEndpoint)
     E2E_PREVIEW_EVENT_HUB_ENDPOINT: $(TestPreviewEventHubCompatibleEndpoint)
     E2E_IOT_HUB_CONNECTION_STRING: $(TestIotHubConnectionString)
+    E2E_IOT_HUB_RESOURCE_ID: $(TestIotHubConnectionString)
```

Comment:
Should be `$(TestIotHubResourceId)`

Reply:
Good catch. E2E test is running now so I will restart it. Luckily this yaml change doesn't require an image rebuild :)